hydra-node/test/Hydra/LoggingSpec.hs

```diff
@@ -24,14 +23,13 @@ spec = do
 
     captured `shouldContain` "{\"foo\":42}"
 
-  aroundAll withJsonSpecifications $ do
-    specify "HydraLog" $ \dir -> do
-      property $
-        withMaxSuccess 1 $
-          conjoin
-            [ prop_validateToJSON @(Envelope (HydraLog Tx ())) (dir </> "logs.yaml") "messages" (dir </> "HydraLog")
-            , prop_specIsComplete @(HydraLog Tx ()) (dir </> "logs.yaml") apiSpecificationSelector
-            ]
+  prop "HydraLog" $
+    property $
+      withMaxSuccess 1 $
+        conjoin
+          [ prop_validateToJSON @(Envelope (HydraLog Tx ())) "logs" "messages" "HydraLog"
+          , prop_specIsComplete @(HydraLog Tx ()) "logs" apiSpecificationSelector
```

Comment:
Should: not request half file names. `"logs"` will be expanded to `logs.yaml` inside this function and this is weird. If we would keep `"logs.yaml"` as before then it is clear(er) that the file in our working tree is meant.